src/gui/providers/gdal/qgsgdalsourceselect.h

```diff
@@ -21,6 +21,7 @@
 #include "qgsabstractdatasourcewidget.h"
 #include "qgis_gui.h"
 #include "qgis_sip.h"
+#include "qgshelp.h"
```

Comment:
These should all be moved to the .cpp files -- minimising the number of includes in the headers greatly speeds up compilation times.